ledger/src/blockstore_db.rs

```diff
@@ -985,23 +985,23 @@ impl TypedColumn for columns::ErasureMeta {
     type Type = blockstore_meta::ErasureMeta;
 }
 
-#[derive(Debug, Clone)]
```

Comment:
Turns out we also don't need `Clone` for this one as well.  This is also removed in this PR.